documentation/ipi-install/modules/ipi-install-cleaning-up-previous.adoc

```diff
@@ -0,0 +1,26 @@
+[id="cleaning-up-previous_{context}"]
+= Cleaning up previous deployments (Optional)
+
+In case you're attempting a new deployment when existing ones were there, there are some steps to take into consideration.
```

Comment:
Let's change to:

In the event there was a previous failed attempted of deployment, run the following steps in order to remove any old artifacts from the failed attempt.

Reply:
in next push